automation/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/NotebookPage.scala

```diff
@@ -97,15 +97,15 @@ class NotebookPage(override val url: String)(override implicit val authToken: Au
     find(kernelNotification).exists { e => e.text == "No kernel" }
   }
 
-  // is the kernel ready?
-  def isKernelReady: Boolean = {
```

Comment:
replaced with `isKernelIdle()`. idle kernel is ready kernel. text `kernel ready` is not reliable way to check for kernel ready. sometimes text is displayed when kernel icon is `busy`